docs/version0.37.md

```diff
@@ -30,8 +30,9 @@ The following new features and notable changes since version 0.36.x are included
 - [AIX is now built on AIX 7.2 TL5](#aix-is-now-built-on-aix-72-tl5)
 - ![Start of content that applies to Java 19 plus](cr/java19plus.png) [Linux&reg; reference compiler updated to gcc 11.2](#linux-reference-compiler-updated-to-gcc-112)
 - [Support added for the `com.sun.management.ThreadMXBean` interface](#support-added-for-the-comsunmanagementthreadmxbean-interface)
-- ![Start of content that applies to Java 11 (LTS) and later](cr/java11plus.png) [Support for PKCS#11 token labels added on z/OS&reg; and Linux on IBM Z&reg;](#support-for-pkcs11-token-labels-added-on-zos-and-linux-on-ibm-z)
+- ![Start of content that applies to Java 11 (LTS) and later](cr/java11plus.png) [Support for PKCS#11 token labels added on Linux on IBM Z&reg;](#support-for-pkcs11-token-labels-added-on-linux-on-ibm-z)
```

Comment:
This topic relates to changes that were made in 0.37.0 release. The token label mentioned here was already existing on z/OS way before this release. In this particular release, the token label was added on Linux on IBM Z as well (I checked with Bob). Because of this, I removed z/OS from this topic only.
We have mentioned that it exists on both z/OS and Linux on IBM Z in the related topic (https://www.eclipse.org/openj9/docs/enhancementstoopenjdksecurity/#support-for-pkcs11-token-labels).

Reply:
Actually it's already qualified as `[Start of content that applies to Java 11 (LTS) and later]`.
The token label support that already exists is only for z/OS IBM Java 8.

Reply:
Ah, now I realize you are correct, the token label support was already there on z/OS Java 11 for some time. I will resolve this.